test/horizontal_blocked_scroll_physics_test.dart

```diff
@@ -105,25 +105,25 @@ MovementResult moveRightNotInLeftRange(HorizontalBlockedScrollPhysics hs) {
 void main() {
   group('HorizontalBlockScrollPhysics', () {
     test('won\'t block by default', () {
-      var hs = HorizontalBlockedScrollPhysics();
+      const hs = HorizontalBlockedScrollPhysics();
```

Comment:
applying recommendations from `flutter_lints` so `flutter analyze` does not complain